### ISSUE TITLE
Any plans to add LargeVis algorithm?

### ISSUE THREAD

**Issue by campbx** [clementfarabet/manifold#13]
It seems to perform better than t-SNE:

Paper: https://arxiv.org/abs/1602.00370
R Implementation: https://github.com/elbamos/largeVis

Thanks!

**Comment by lvdmaaten** [comment]
No, but I have been working on a much faster t-SNE implementation. It is unclear to me whether the speed differences in the LargeVis paper are due to algorithmic changes or due to better implementation. The current BH t-SNE implementation is pretty bare bone (simple but inefficient tree implementations, no OpenMP support, etc.) so it can be sped up very substantially.
